supervisor/dbus/network/const.py

```diff
@@ -12,17 +12,19 @@
 ATTR_ADDRESS_DATA = "AddressData"
 ATTR_CONFIGURATION = "Configuration"
 ATTR_CONNECTION = "Connection"
+ATTR_CONNECTIONS = "connections"
 ATTR_DEFAULT = "Default"
 ATTR_DOMAINS = "domains"
 ATTR_FILENAME = "filename"
 ATTR_FLAGS = "flags"
-ATTR_GATEWAY = "gateway"
+ATTR_GATEWAY = "Gateway"
```

Comment:
we use lowercase for our API

Reply:
Yeah, this was fast and dirty to get something working.
I will move attributes used for the api to supervisor/const.py (or maybe create a new one under supervisor/api/const.py), and have attrs for dbus responses here.